src/contact.js

```diff
@@ -0,0 +1,25 @@
+const contact = document.createElement('div');
+
+const paragraph = document.createElement('p');
+paragraph.textContent = 'You can conatct us in:';
+contact.appendChild(paragraph);
+
+
+const contacts = [
+  'Mail: youcefabdellani@gmail.com',
+  'facebook: youcefabdellani',
+  'official site: youcefrestaurant.dz',
+];
+
+const ulist = document.createElement('ul');
+for (let i = 0; i < contacts.length; i += 1) {
+  const item = document.createElement('li');
+  item.textContent = contacts[i];
+  ulist.appendChild(item);
+}
```

Comment:
- [ ] Please aim to use ES6 syntax more. Kindly replace this forLoop with a map function:
`contacts.map((item, index) => { code here })`